runtime/doc/eval.txt

```diff
@@ -3507,9 +3507,22 @@ getcurpos()	Get the position of the cursor.  This is like getpos('.'), but
 			MoveTheCursorAround
 			call setpos('.', save_cursor)
 <
-							*getcwd()*
-getcwd()	The result is a String, which is the name of the current
-		working directory.
+getcwd([{window}[, {tab}]])					*getcwd()*
+		With no argument the result is a String, which is the name of the
+		current effective working directory. If an arguments are provided for a
+		window or a tabpage the working directory of that scope is returned.
+
+		Tabs and windows are identified using their respective number. If a
+		number is 0 the current tab or window is implied. If one is not given
+		it defaults to 0. Thus
+		
+			getcwd()
+			getcwd(0)
+			getcwd(0, 0)
+
+		are all equal. If {window} is -1 it will be ignored and only the tab
+		will be looked at.
```

Comment:
Yes, we dropped that in favour of window/tab numbers because Vim was going to change `getcwd()` as well: the user could pass the number of a window as an argument to get the working directory of that window, so I did the same and extended it by one optional parameter for the level above (tabs).